### DISCUSSION TITLE
Do something about AppGet, which is now defunct

### DISCUSSION THREAD

**Discussion by StewAlexander-com** [microsoft/winget-cli#518] [General]
Just a comment to do something about AppGet which is now defunct; we and plenty of others used AppGet, and with it down, and WinGet not up, we are left with Chocolatey which doesn't work for us due to how it does things...  we can get it to work, and we have, but AppGet was a much smoother easier alternative.

Thanks ...

**Comment by holtalanm** [comment]
> we are left with Chocolatey

I've just been using Scoop until WinGet is up to speed.  Works really well, imo. 

https://scoop.sh/

**Comment by denelon** [comment]
I've just enabled GitHub Discussions. I'm converting this Issue to a Discussion.

**Comment by superusercode** [comment]
I suppose the ideal solution would be to go through their manifests list at https://github.com/appget/appget.packages and make manifests for anything that was missed.

Some of the manifests depend on zip support #140

**Reply by StewAlexander-com** [reply]
That would be nice for us in the community that will probably at some point use the Windows product
